core/data/src/main/java/com/moya/funch/datasource/remote/RemoteMatchDataSourceImpl.kt

```diff
@@ -19,7 +19,16 @@ class RemoteMatchDataSourceImpl @Inject constructor(
                     targetCode = targetCode
                 )
             )
-        }.mapCatching { it.data }
+        }.mapCatching {
+            saveMbtiCollection(it.data.profile.mbti)
+            it.data
+        }
+    }
+
+    private fun saveMbtiCollection(mbti: String) {
+        dataStore.mbtiCollection = dataStore.mbtiCollection.toMutableSet().apply {
+            add(mbti)
+        }.toSet()
```

Comment:
```suggestion
    private fun saveMbtiCollection(mbti: String) {
        dataStore.mbtiCollection = dataStore.mbtiCollection + mbti
```

plus opeerator를 사용하는게 더 간결할 거 같네용